Formula/lilv.rb

```diff
@@ -24,4 +24,9 @@ def install
     system "./waf"
     system "./waf", "install"
   end
+
+  test do
+    lv2info --version
```

Comment:
You're still in ruby, so these need to be surrounded with `shell_output` and then tested with `assert_equal` to ensure we're getting something reasonable from them.

See `Formula/azure-cli.rb.rb` as an example.